src/repos/EstablishmentRepo.mjs

```diff
@@ -1,5 +1,54 @@
+import sqlite3 from 'sqlite3';
+import {pathDbFromRepos, connect} from '../../database/index.js';
+import { Establishment } from '../models/index.mjs'
+
 export class EstablishmentRepo {
-    async createEstablishment(name, estType) {}
-    async getEstablishment(estId) {}
+
+    constructor() {
+        this.pathDB = pathDbFromRepos;
+        this.DB = connect(this.pathDB);
+    }
+
+    async createEstablishment(email, password, prefixPhoneNumber, phoneNumber, contactEmail, name, estType) {
+        let query = 'INSERT INTO ESTABLISHMENT (email, password, prefixPhoneNumber, phoneNumber, contactEmail, name, estType) VALUES (?, ?, ?, ?, ?, ?, ?)';
+        this.DB.run(query, [email, password, prefixPhoneNumber, phoneNumber, contactEmail, name, estType], (err)=>{
+            if(err){
```

Comment:
You should propagate the error to the caller, otherwise the API server has no idea if the insertion completed successfully or not